wrappers/swift/DiagnosticDataViewer.swift

```diff
@@ -0,0 +1,85 @@
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/// Wrapper class over `ODWDiagnosticDataViewer` representing Diagnostic Data Viewer Hook.
+public class DiagnosticDataViewer {
+
+    /// MARK: Initialization Methods
+
+    /**
+    Initializes Data Viewer with a specified machine identifier.
+
+    - Parameters:
+        - machineIdentifier: `String` that contains the machine identifier.
+    */
+    public static func initViewerWithMachineIdentifier(machineIdentifier: String) {
+        ODWDiagnosticDataViewer.initializeViewer(withMachineIdentifier: machineIdentifier)
+    }
+
+    /// MARK: Behavior Methods
+
+    /**
+    Enables Data Viewer.
+
+    - Parameters:
+        - endpoint: A `String` that contains endpoint to route events.
+        - completionWithResult: Code to execute when enable is completed.
+            - Note: This value can be null.
+    
+    Execution is dispatched to queue with default priority.
+    */
+    public static func enableRemoteViewer(endpoint: String, completionWithResult completion: @escaping (Bool) -> Void) {
+        ODWDiagnosticDataViewer.enableRemoteViewer(endpoint, completionWithResult: completion)
+    }
+
+    /**
+    Enables Data Viewer.
+
+    - Parameters:
+        - endpoint: A `String` that contains endpoint to route events.
+    */
+    public static func enableRemoteViewer(endpoint: String) -> Bool {
+        return ODWDiagnosticDataViewer.enableRemoteViewer(endpoint)
+    }
+
+    /**
+    Disables Data Viewer.
+
+    - Parameters:
+        - completionWithResult: Code to execute when disable is completed.
+            - Note: This value can be null.
+
+    Execution is dispatched to queue with default priority.
+    */
+    public static func disableViewer(completionWithResult completion: @escaping (Bool) -> Void) {
+        ODWDiagnosticDataViewer.disableViewer(completion)
+    }
+
+    /// Disables Data Viewer.
+    public static func disableViewer() -> Bool {
```

Comment:
The Swifty way of writing getters and setters would be as computed variables; it would be nice for us to leverage that pattern here and in the other PRs :)

Reply:
can you please explain this one? as i see, only viewEnabled can be extracted to computed properties and rest of the viewer methods would need to be as methods.

Reply:
My mistake. I meant to put this one function lower.

```
    public static var viewerEnabled: Bool {
         ODWDiagnosticDataViewer.viewerEnabled()
     }

     public static var currentEndpoint: String {
         return ODWDiagnosticDataViewer.currentEndpoint()
     }
```

Reply:
I see, but there are other related methods with these properties which can't be combined into setter computed properties, such as disableViewer.
Converting the just one part of the property into computed ones does not seem correct to me.

Reply:
`disableViewer` is a function performing a task, as opposed to a variable holding state. Variables, whether computed or stored, tend to communicate the storage of state in Swift, which functions communicate the performance of an action.